examples/2d/mesh2d_depth.rs

```diff
@@ -0,0 +1,90 @@
+//! Used to test transforms with mesh2d
```

Comment:
Maybe also specify that it's testing the depth / depth buffer and what you should expect to see

Reply:
Ah, yeah, sorry, I wrote that just so clippy would stop complaining and forgot to write an actual description once I was done with it 😅